src/api/requirements.txt

```diff
@@ -1,5 +1,17 @@
-fastapi~=0.63.0
-uvicorn~=0.13.4
-orm~=0.1.5
-asyncpg~=0.22.0
-sqlalchemy[async]~=1.4.7
\ No newline at end of file
+aiosqlite==0.17.0
+asyncpg==0.22.0
+click==7.1.2
+databases==0.4.3
+fastapi==0.63.0
+greenlet==1.0.0
+h11==0.12.0
+importlib-metadata==3.10.1
+orm==0.1.5
```

Comment:
I have removed the usage of orm,
Can you remove this and other relevant packages it uses.

Reply:
This isn't massively important

Reply:
Sure, I'll do that

Reply:
I've done the needful.